src/main/java/com/napier/sem/DatabaseConnectionImpl.java

```diff
@@ -0,0 +1,140 @@
+package com.napier.sem;
```

Comment:
This class name is Hungarian, we want to avoid using this see https://stackoverflow.com/questions/111933/why-shouldnt-i-use-hungarian-notation. I believe the naming convention is to name the interface with a prefix I (upper case eye). This is also slightly hungarian but imo its better